loops.js

```diff
@@ -51,3 +76,15 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 111; i++) {
+    if (i < 3) {
+        console.log(i);
+    }
+    if (i >= 3 && i < 6){
+        console.log('> ' + i);
+    }
+    if (i > 6 && i < 10){
+        console.log('>> ' + i);
+    }
+}
```

Comment:
Hier gebruik je 3x een if statement, terwijl je beter een if/elsif/else kunt gebruiken. Dat is overzichtelijker voor je mede developer en voor jou tijdens het teruglezen! 

Wel leuk weer die (i < 111;) in regel 79... Probeer je me scherp te houden..? hahaha!